.gitignore

```diff
@@ -7,7 +7,7 @@ bazel-genfiles
 bazel-grpc-gateway
 bazel-out
 bazel-testlogs
-.bazelrc
+/.bazelrc
```

Comment:
I accidentally made .bazelrc files in subdirectories and this check is ignoring any .bazelrc even if its made in places other than the project root.